tests/components/braviatv/test_config_flow.py

```diff
@@ -0,0 +1,248 @@
+"""Define tests for the Bravia TV config flow."""
+from collections import OrderedDict
+
+from asynctest import patch
+
+from homeassistant import data_entry_flow
+from homeassistant.components.braviatv.const import CONF_IGNORED_SOURCES, DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_HOST, CONF_PIN
+
+from tests.common import MockConfigEntry
+
+BRAVIA_SYSTEM_INFO = {
+    "product": "TV",
+    "region": "XEU",
+    "language": "pol",
+    "model": "TV-Model",
+    "serial": "serial_number",
+    "macAddr": "AA:BB:CC:DD:EE:FF",
+    "name": "BRAVIA",
+    "generation": "5.2.0",
+    "area": "POL",
+    "cid": "very_unique_string",
+}
+
+BRAVIA_SOURCE_LIST = OrderedDict(
+    [
+        ("HDMI 1", "extInput:hdmi?port=1"),
+        ("HDMI 2", "extInput:hdmi?port=2"),
+        ("HDMI 3/ARC", "extInput:hdmi?port=3"),
+        ("HDMI 4", "extInput:hdmi?port=4"),
+        ("AV/Component", "extInput:component?port=1"),
+    ]
+)
+
+CONFIG = {CONF_HOST: "bravia-host", CONF_PIN: "1234"}
+
+config_entry = MockConfigEntry(
```

Comment:
This should not be defined at the module level. Create a new instance for each test. The instance and it's context can not be considered constant.